contracts/middle-layer/resource-mirror/storage/GroupStorage.sol

```diff
@@ -34,6 +34,7 @@ contract GroupStorage is CmnStorage {
         UpdateGroupOpType opType;
         address[] members;
         bytes extraData; // abi.encode of ExtraData
+        uint64[] memberExpiration; // timestamp(UNIX) of member expiration
```

Comment:
Do we check the length of `memberExpiration` should equal to length of `members`?

Reply:
Yes, I think so. We need check that in `updateGroup()`

Reply:
added